CopilotKit/packages/runtime/src/lib/runtime/copilot-runtime.ts

```diff
@@ -975,46 +975,57 @@ please use an LLM adapter instead.`,
       if (!aguiAgent) {
         throw new Error(`Agent: ${agent.name} could not be resolved`);
       }
-      // @ts-expect-error -- both clients are the same
-      client = aguiAgent.client;
+      if ("client" in aguiAgent) {
+        client = aguiAgent.client as unknown as LangGraphClient;
+      }
     }
     let state: any = {};
-    try {
-      state = (await client.threads.getState(threadId)).values as any;
-    } catch (error) {
-      // All errors from agent state loading are user configuration issues
-      const errorMessage = error instanceof Error ? error.message : String(error);
+    if (client) {
+      try {
+        state = (await client.threads.getState(threadId)).values as any;
```

Comment:
why do we need to fetch the state? can't we go for a full ignore solution?

Instead let's just return an empty state